ldms/src/contrib/store/Makefile.am

```diff
@@ -10,6 +10,6 @@ do_subst = @LDMS_SUBST_RULE@
 	$(do_subst) < $< > $@
 	chmod 755 $@
 
-if ENABLE_IPMIREADER
-SUBDIRS += ipmireader
+if ENABLE_TUTORIAL_STORE
+SUBDIRS += tutorial
 endif
```

Comment:
Use new "MAYBE" pattern.

Reply:
I have changed these in the files related to contrib. I have held off doing this in other Makefile.am's since I wanted to localize the change. I can do these all in another change if that is the generally accepted route. @morrone @baallan @tom95858 @narategithub

Reply:
Right, I was only suggesting you change the ones introduced by your commit, not all of the ones in the source tree.  I have a PR already out there to fix up most of the other locations in the tree.

Reply:
updated in commit